functions/index.js

```diff
@@ -51,12 +51,12 @@ const common_functions = {
 
     /**
      * Check whether string is relative path or not
-     * @param {String} path 
-     * @returns 
+     * @param {String} path
+     * @returns
      */
     isPathRelative(path) {
-        if(typeof path !== 'string') return false 
-        let isAbsolute = /^(?:\/|[a-z]+:\/\/)/.test(path);
+        if(typeof path !== 'string') return false
+        let isAbsolute = /^([a-zA-Z]\:|\\\\[^\/\\:*?"<>|]+\\[^\/\\:*?"<>|]+)(\\[^\/\\:*?"<>|]+)+(\.[^\/\\:*?"<>|]+)$/.test(path);

```

Comment:
Do we need this change even though we changed the order of `if ... else...` in another file?

Reply:
Yes, we need it.